packages/admin/src/Charcoal/Admin/ServiceProvider/AdminServiceProvider.php

```diff
@@ -203,6 +206,31 @@ protected function registerAdminServices(Container $container)
         });
     }
 
+    /**
+     * @param Container $container A Pimple DI container.
+     * @return void
+     */
+    protected function registerModelDependencies(Container $container)
+    {
+        /**
+         * @param Container $container A Pimple DI container.
+         * @return array The model dependencies array.
+         */
+        $container->extend('model/dependencies', function (array $modelDependencies, Container $container) {
+            $modelDependencies['view'] = $container['admin/view'];
+            return $modelDependencies;
+        });
```

Comment:
Previously, the `view` service was extended to replace the view engine with Mustache.

Extending `model/dependencies` to replace the `view` parameter might just be moving the initial app/admin view rendering problem here.